### PR TITLE
Update CRIU version in the shell Dockerfile

### PR DESCRIPTION
**Pull request by boucher**
CRIU needs a newer version to work with the updates pulled in by containerd 1.0

Signed-off-by: Ross Boucher <rboucher@gmail.com>

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM

ping @kolyshkin PTAL

**Comment by kolyshkin** [comment]
@boucher can you please also
* remove `-v` option from tar invocation;
* remove `apt-get install libnet-dev` (and the preceding comment) and add `libnet-dev` to the list at the beginning of the file (where all the other criu deps are living anyway)

**Review by kolyshkin — CHANGES_REQUESTED** [review]
@boucher can you please also
* remove `-v` option from tar invocation;
* remove `apt-get install libnet-dev` (and the preceding comment) and add `libnet-dev` to the list at the beginning of the file (where all the other criu deps are living anyway)

File: Dockerfile [hunk 1]
```diff
@@ -194,4 +194,3 @@ ENV GOMETALINTER_OPTS="--deadline 2m"
 
 # Upload docker source
 COPY . /go/src/github.com/docker/docker
-
```
Comment (kolyshkin) [inline Dockerfile]: remove this hunk

**Review by kolyshkin — CHANGES_REQUESTED** [review]

**Comment by boucher** [comment]
Done

**Comment by kolyshkin** [comment]
LGTMThank you, Ross!

**Comment by kolyshkin** [comment]
@boucher I am very sorry, totally forgot it that we moved github repo to https://github.com/checkpoint-restore/criu/; can you please change the download URL accordingly?

**Comment by kolyshkin** [comment]
This might also fix https://github.com/moby/moby/issues/35235

**Review by kolyshkin — APPROVED** [review]
LGTM

**Review by yongtang — APPROVED** [review]
LGTM

**Pull request merged**